advanced/creating-a-skin.md

```diff
@@ -4,48 +4,56 @@ title: Create a skin for TinyMCE
+You now have the development environment set up and are ready to get to work.
 
-> Note: This is the same directory that contains the "light gray" skin.
+## Making or editing a skin
 
-Issue the `grunt less` command from your console. This command automatically builds CSS files out of the `less` files contained in the skins directories. Use `grunt watch` to automatically build the skin when changes are made to the `less` files.
+Make sure you have performed the preparation step above.
 
-The build process will produce these CSS files:
-* `skin.min.css`
-* `content.min.css`
-* `content.inline.min.css`
+### Orientation
 
-`content.min.css` and `content.inline.min.css` are used for content inside the editor. These files contain CSS for normalizing editing behavior and the appearance of things like video placeholders.
+Begin by looking in the `src/less/skins/` folder where you find two folders: `ui` which is the skins for the editor and `content` which skins the content within the editor. The skin imports the theme `less` files located in `src/less/theme/`. You are not supposed to edit the CSS in the theme folder, instead we have created variables for relevant css properties such as colors, margins and spacings which you use in your skin file to customize TinyMCE. This have multiple benefits: you can do quite advanced changes and still easily update TinyMCE since the only thing you have changed is a variable file, and you benefit from all the browser testing we make.
 
-`skin.min.css` are used to render the UI.
+> Note: The skin **only** changes the visual presentation of the UI and **not** the placement of elements. Placement of elements is done by the TinyMCE UI framework. This framework makes it possible to do complex UI layouts on all browsers without touching any CSS when plugins are created.
 
-## Modifying the less files
+### Creating a skin
 
-Most CSS elements can be modified by altering the `variables.less` file. This file contains variables such as:
-* font size
-* font family
-* colors
-* borders
+1. Begin by duplicating the `default` folder located in `src/less/skins/ui/` and rename it to the name of your skin.
+2. Open the file `src/less/theme/globals/global-variables.less` and **copy** the variables you like to change to your `skin.less` file in the folder you duplicated in the previous step. Change the values. The variables you put in `skin.less` will override the default values.
+3. For more detailed customizations, review the variables in each component, such as `src/less/theme/components/toolbar-button.less` and copy the ones you want to change to `skin.less`.
+4. To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` depending on if you are using the inline mode or not.
+5. Preview your skin by building it using the `gulp` command in your terminal.
```

Comment:
This is looking good! Could we add something about _how_ to preview it though? I'm not sure if you mean by looking at oxide demos or by copying the built CSS files into TinyMCE.

Reply:
Also, running gulp by itself will only work if people have gulp installed globally. Since gulp is a dependency of the oxide project, they could also use `npx gulp`.

Reply:
I have added a paragraph about opening a browser and pointing it to localhost.xxxx